src/NUnitFramework/TestBuilder.cs

```diff
@@ -179,7 +179,7 @@ public static ITestResult ExecuteWorkItem(WorkItem work)
             // TODO: Replace with an event - but not while method is static
             while (work.State != WorkItemState.Complete)
             {
-#if PORTABLE || NETSTANDARD1_6
+#if PORTABLE
                 System.Threading.Tasks.Task.Delay(1);
 #else
                 Thread.Sleep(1);
```

Comment:
In general, do we think it's better to use `Thread.Sleep` or `Task.Delay` where both are available? I'm wondering if we should change this generally in a future PR.